src/SignaturePad.InkPresenter.Shared/InkPresenter.cs

```diff
@@ -143,14 +164,26 @@ private bool AddStrokeInternal (IEnumerable<NativePoint> points, NativeColor col
 			}
 
 			var newpath = new NativePath ();
+#if NET471
+			
+			var prev = new NativePoint (strokePoints[0].X, strokePoints[0].Y);
+			foreach (var point in strokePoints.Skip (1))
+			{
+				var destPoint = new NativePoint (point.X, point.Y);
+				newpath.AddLine(prev, destPoint);
+				prev = destPoint;
+			}
+
+			paths.Add (new InkStroke (newpath, strokePoints, color, width));
+#else
```

Comment:
Can we use extension methods here as well? See the other platforms: https://github.com/xamarin/SignaturePad/blob/master/src/SignaturePad.Shared/Extensions.cs

All smoke and mirrors, but I would like to keep these files clean of `#if` so it is easier to modify and understand.

Reply:
+